src/components/DataSubmissions/DataUpload.tsx

```diff
@@ -1,386 +1,142 @@
-import { CREATE_BATCH, CreateBatchResp, UPDATE_BATCH, UpdateBatchResp } from "../../graphql";
-import { useAuthContext } from "../Contexts/AuthContext";
-import DeleteDialog from "../../content/dataSubmissions/DeleteDialog";
+import { FC, ReactElement, useMemo, useState } from "react";
+import { useLazyQuery } from "@apollo/client";
+import { useSnackbar } from "notistack";
+import { Box, Button, Typography, styled } from "@mui/material";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import env from "../../env";
+import { RETRIEVE_CLI_CONFIG, RetrieveCLIConfigResp } from "../../graphql";
+import { downloadBlob, filterAlphaNumeric } from "../../utils";
 import FlowWrapper from "./FlowWrapper";
+import UploaderToolDialog from "../UploaderToolDialog";
+import UploaderConfigDialog, { InputForm } from "../UploaderConfigDialog";
 
-const StyledUploadTypeText = styled(Typography)(() => ({
-  color: "#083A50",
-  fontFamily: "'Nunito', 'Rubik', sans-serif",
-  fontSize: "16px",
-  fontStyle: "normal",
-  fontWeight: 700,
-  lineHeight: "19.6px",
-}));
+export type Props = {
+  /**
+   * The submission to download a pre-configured CLI config for.
+   */
+  submission: Submission;
+};
 
-const StyledMetadataText = styled(StyledUploadTypeText)(() => ({
-  "&.MuiTypography-root": {
-    color: "#000000",
-  },
-}));
+const StyledBox = styled(Box)({
+  maxWidth: "790px",
+  lineHeight: "22px",
+  marginTop: "10px",
+});
 
-const StyledUploadFilesButton = styled(Button)(() => ({
-  minWidth: "137px",
-  padding: "10px",
-  color: "#FFF",
-  fontFamily: "'Nunito'",
-  fontSize: "16px",
+const StyledDownloadButton = styled(Button)({
+  padding: "10px 6px",
+  fontSize: "14px",
   fontStyle: "normal",
-  lineHeight: "24px",
+  lineHeight: "16px",
   letterSpacing: "0.32px",
-  textTransform: "none",
+  width: "137px",
+  height: "44px",
   "&.MuiButtonBase-root": {
     marginLeft: "auto",
-    minWidth: "137px",
-  },
-}));
-
-const StyledChooseFilesButton = styled(LoadingButton)(() => ({
-  minWidth: "137px",
-  padding: "10px",
-  fontFamily: "'Nunito', 'Rubik', sans-serif",
-  fontSize: "16px",
-  fontStyle: "normal",
-  fontWeight: 500,
-  lineHeight: "24px",
-  textTransform: "initial",
-  marginLeft: "12px",
-  marginRight: "12px",
-  "&.MuiButtonBase-root": {
-    marginLeft: "15px",
   },
-}));
+});
 
-const StyledFilesSelected = styled(Typography)(() => ({
-  color: "#083A50",
-  fontFamily: "'Nunito', 'Rubik', sans-serif",
-  fontSize: "16px",
-  fontStyle: "italic",
-  fontWeight: 400,
-  lineHeight: "19.6px",
-  minWidth: "135px",
-}));
-
-const StyledUploadActionWrapper = styled(Stack)(() => ({
-  "&.MuiStack-root": {
-    justifyContent: "center",
-    alignItems: "center",
-    marginLeft: "20px",
+const StyledToolButton = styled(Typography)({
+  color: "#005999",
+  cursor: "pointer",
+  lineHeight: "16px",
+  fontWeight: 700,
+  display: "inline",
+  "& span": {
+    borderBottom: "1px solid #D1D1D1",
```

Comment:
Small thing I noticed, underline and icon seems a bit off from the design. The icon is not a big deal so we can disregard that if you prefer. 

Figma:
![Screenshot 2024-04-30 at 4 05 33 PM](https://github.com/CBIIT/crdc-datahub-ui/assets/70384003/9d516917-6eff-4eed-9cb6-a094d9464d62)

Actual:
![Screenshot 2024-04-30 at 4 08 10 PM](https://github.com/CBIIT/crdc-datahub-ui/assets/70384003/cc50a2eb-f1dc-4d1e-b4fb-5e85a456900e)

Reply:
Good catch. I fixed the underline, it should be much closer to the Figma styling now.  a6b7266

I left the icon as the one from MUI since the difference is relatively minor and it's one less asset to manage.